data/hails.txt

```diff
@@ -2030,7 +2030,273 @@ phrase "friendly author"
 
 phrase "friendly disabled"
 	word
-		"Our ship has been disabled! Please come board our ship and patch us up!"
+		"Our ship has been disabled! "
+		"We've gotten ourselves in a bit of a situation! "
+		"We've lost control of our ship! "
+		"Our ship took quite the pounding! "
+		"Hey! "
+		"S.O.S! S.O.S! "
+	word
+		"Please come board our ship and patch us up!"
+		"We would appreciate any help getting us running again!"
+		"We can't fix this on our own!"
+		"Could you come over and help us please?"
+		"Requesting emergency repairs!"
+		
+phrase "friendly disabled"
+	word
+		"Hey! "
+		""
+		""
+		"You're a friendly, right? "
+		"You answered! "
+		"Good to see a friendly face! "
+		""
+		"Hey, pleaase! "
+		""
+		""
+	word
+		"Our ship is disabled, "
+		"We're kinda stuck here, "
+		"We're in a bit of a bind here, "
+		"We're stuck drifting around in this deathtrap, "
+		"Not to be a bother, but "
+		"Could you come over, and "
+	word
+		"could you lend a hand"
+		"can you get us back on our feet"
+		"would you help us"
+		"any chance you could get us up and running again"
+		"could you render assistance"
+		"can we get a little help here"
+	word
+		"?"
+		" please?"
+		"? I don't wanna get stranded here!"
+		" pretty please?"
+		" so we can get out of here?"
+		" as soon as possible?"
+		"?"
+		"?"
```

Comment:
There should be a newline from here to the next phrase.
It would also be cleaner faster to move all of these question marks:
```
    word
        "?"
        "?"
        "?"
        "?"
        "?"
        "?"
        "? I don't wanna get stranded here!"